app/templates/frameworks/_agreement_returned_legal.html

```diff
@@ -0,0 +1,106 @@
+{% if supplier_is_on_framework and framework.status in ['standstill', 'live'] and supplier_framework.agreementReturned %}
+  <div>
+    <h2 class="page-subheading">Legal documents</h2>
+    <div class="dmspeak">
+    {% if supplier_framework.countersigned %}
+      <p>
+        Your framework agreement signature page has been sent to
+        the Crown Commercial Service (<abbr title="Crown Commercial Service">CCS</abbr>).
+        They’ll check and countersign the agreement, or contact you if there’s a
+        problem. You don’t need to wait for <abbr title="Crown Commercial Service">CCS</abbr> to
+        countersign your agreement before you start selling services.
+      <p/>
+    {% else %}
+      <p>
+        Your original and counterpart signature pages, and the standard framework
+        agreement, form a complete, legal contract.
+      </p>
+    {% endif %}
+    </div>
+
+  {% if framework.frameworkAgreementVersion %}
+    {% import "toolkit/summary-table.html" as summary %}
+    {% call(item) summary.mapping_table(
+      caption='Agreement details',
+      field_headings=[
+        'Label',
+        'Value'
+      ],
+      field_headings_visible=False
+    ) %}
+      {% call summary.row() %}
+        {{ summary.field_name('Person who signed') }}
+        {% call summary.field() %}
+          <p>{{ supplier_framework.agreementDetails.signerName }}</p>
+          <p>{{ supplier_framework.agreementDetails.signerRole }}</p>
+        {% endcall %}
+      {% endcall %}
+      {% call summary.row() %}
+        {{ summary.field_name('Submitted by') }}
+        {% call summary.field() %}
+          <p>{{ supplier_framework.agreementDetails.uploaderUserName }}</p>
+          <p>{{ supplier_framework.agreementDetails.uploaderUserEmail }}</p>
+          <p>{{ supplier_framework.agreementReturnedAt|datetimeformat }}</p>
+        {% endcall %}
+      {% endcall %}
+      {% call summary.row() %}
+        {{ summary.field_name('Countersignature') }}
+        {{ summary.field_name('Waiting for CCS to countersign') }}
```

Comment:
Actually according to the prototype this information table should just be removed entirely when countersigned.